apps/server/src/test-adapter/constants.ts

```diff
@@ -10,6 +10,10 @@ export const TEST_EXAMPLE_C_LABEL_TEXT = "TestExampleC Label";
 
 export const testAggregatorMemberGuid = "testAggregatorMemberGuid";
 
+export const testDataValidatorStartTimeError =
```

Comment:
These are only used once and are test specific right? I'd probably hardcode them in the tests instead

Reply:
I'm actually using them in the test-adapter as return values for the DataRequestValidation.transactions return value, so I'm using them in two places each.